examples/toys/repo.py

```diff
@@ -1,4 +1,11 @@
+import os
+import sys
 from dagster import RepositoryDefinition
+from dagster.utils import script_relative_path
+
+sys.path.insert(0, os.path.abspath(script_relative_path('.')))
```

Comment:
@mgasner this is required to run this from the command line.

@mgasner do you have a sense why the cmdline needs this but dagit does not?

Reply:
what do you mean to run this from the command line? `python repo.py` works fine for me..

Reply:
ideally we figure this out and can remove this before landing

Reply:
alright i think i just hit this, the repro is 
`dagster pipeline execute error_monster` from inside `examples/toys`

Reply:
https://gist.github.com/alangenfeld/9cfa3650109901a39d44fd2e942ea80f